site/src/pages/ManagementSettingsPage/OrganizationSettingsPage.tsx

```diff
@@ -26,32 +29,88 @@ const OrganizationSettingsPage: FC = () => {
     deleteOrganization(queryClient),
   );
 
-  const org = organizationName
-    ? getOrganizationByName(organizations, organizationName)
-    : getOrganizationByDefault(organizations);
+  // TODO: If we could query permissions based on the name then we would not
+  //       have to cascade off the organizations query.
+  const organization =
+    organizations &&
+    organizationName &&
+    getOrganizationByName(organizations, organizationName);
+  const permissionsQuery = useQuery(
+    organization
+      ? organizationPermissions(organization.id)
+      : { enabled: false },
+  );
```

Comment:
Maybe we could wrap this into a function like `getOrgPermissionsByName` since it is being used in a few places?

Reply:
Yeah I could probably just bake in this behavior straight into `organizationPermissions` actually.